0-bootstrap/main.tf

```diff
@@ -202,32 +199,14 @@ resource "google_folder_iam_member" "folder_cb_sa_browser" {
   member = "serviceAccount:${data.google_project.cloudbuild.number}@cloudbuild.gserviceaccount.com"
 }
 
-resource "google_organization_iam_member" "org_tf_compute_security_policy_admin" {
-  count  = var.parent_folder == "" ? 1 : 0
-  org_id = var.org_id
-  role   = "roles/compute.orgSecurityPolicyAdmin"
-  member = "serviceAccount:${module.seed_bootstrap.terraform_sa_email}"
-}
+module "granular_service_account" {
```

Comment:
Nit: I don't think having this as a module provides alot of value and adds a level of indirection. WDYT about just having this as sa.tf in the root config?

Reply:
fixed